src/renderer/pages/Settings/Networks/components/index.ts

```diff
@@ -1,2 +1,2 @@
-export { CustomRpcModal } from './CustomRpcModal/CustomRpcModal';
+export { AddCustomRpcModal } from '../../../../features/network/CustomRpcForm/ui/AddCustomRpcModal';
```

Comment:
```suggestion
export { AddCustomRpcModal } from '@features/network';
```

Reply:
Indeed, this was a stray change that's not needed.